hsc/crawler.py

```diff
@@ -224,14 +235,14 @@ def get_submissions(self, submissions):
 
 def main():
 	offset = 0
-	limit  = 10 # you should change this
 
 	crawler = Crawler()
-
+	crawler.parse_script()
```

Comment:
I think we should separate input logic from crawler (in a different PR) or else it will become messy with time.

Reply:
sure

Reply:
@rishabhsingh971 If you feel like this should be done in some separate PR, then you can create a new issue from this comment itself.

Reply:
I was waiting for someone's agreement :P